lms/djangoapps/teams/static/teams/js/spec/views/team_profile_spec.js

```diff
@@ -5,26 +5,31 @@ define([
 ], function (_, AjaxHelpers, TeamModel, TeamProfileView, TeamDiscussionSpecHelper, DiscussionSpecHelper) {
     'use strict';
     describe('TeamProfileView', function () {
-        var discussionView, createTeamProfileView;
+        var profileView, createTeamProfileView;
 
         beforeEach(function () {
             DiscussionSpecHelper.setUnderscoreFixtures();
         });
 
-        createTeamProfileView = function(requests) {
+        createTeamProfileView = function(requests, options) {
             var model = new TeamModel(
```

Comment:
What we've done elsewhere is use ``_.extends`` to merge the options into the default settings. This makes it easier to pass any parameters that you need without having to update this method every time.